bindata/network/kuryr/005-daemon.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: kuryr-cni
+  namespace: kuryr
+  annotations:
+    kubernetes.io/description: |
+      This DaemonSet launches the kuryr-daemon component.
+spec:
+  template:
+    metadata:
+      name: kuryr-cni
+      labels:
+        app: kuryr
+        component: network
+        type: infra
+        openshift.io/component: network
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      hostNetwork: true
+      serviceAccountName: kuryr
+      initContainers:
+      - name: install-cni-plugins
+        image: {{ .NodeImage }}
```

Comment:
Yes, that's exactly what we want, this step only installs those generic plugin binaries, mainly loopback that is required for other plugins to work correctly.

kuryr-cni plugin binary is installed by kuryr-cni container entrypoint because it's basically a bash script that executes `runc exec` on kuryr-cni container while passing the CNI_* env vars. Kuryr is a Python application and it's extremely problematic to make it a binary that can be easily moved onto the host. Some more info about what solutions we've explored is here: https://opensource.com/article/18/10/how-kubernetize-openstack-service#injecting-the-kuryr-cni-executable